tiledb/storage_format/serialization/serializers.h

```diff
@@ -142,7 +142,9 @@ class Deserializer {
    */
   Deserializer(const void* data, storage_size_t size)
       : ptr_(static_cast<const uint8_t*>(data))
-      , size_(size) {
+      , size_(size)
```

Comment:
As discussed above we might be able to keep the deserializer unchanged.

Reply:
Deserializer reverted to unchanged